### PR TITLE
Ensure FQCN in annotations.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/TestSuite/Constraint/Response/BodyContains.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ class BodyContains extends ResponseBase
     /**
      * Constructor.
      *
-     * @param Response $response Response
+     * @param \Cake\Http\Response $response \Cake\Http\Response
```
Comment (dereuromark) [inline src/TestSuite/Constraint/Response/BodyContains.php]: Dup error

File: src/TestSuite/Constraint/Response/CookieEquals.php [hunk 2]
```diff
@@ -30,7 +30,7 @@ class CookieEquals extends ResponseBase
     /**
      * Constructor.
      *
-     * @param Response $response Response
+     * @param \Cake\Http\Response $response \Cake\Http\Response
```
Comment (dereuromark) [inline src/TestSuite/Constraint/Response/CookieEquals.php]: No need to repeat IMO.

File: src/TestSuite/Constraint/Response/BodyContains.php (hunk 1 above)
Comment (ADmad) [reply src/TestSuite/Constraint/Response/BodyContains.php]: Seems the sniffer is a bit overzealous and modifies descriptions too. I will open an issue with slevomat standards.

Comment (ADmad) [reply src/TestSuite/Constraint/Response/BodyContains.php]: slevomat/coding-standard#394

File: src/Core/PluginCollection.php [hunk 3]
```diff
@@ -105,7 +105,7 @@ protected function loadConfig(): void
      *
      * @param string $name The plugin name to locate a path for. Will return '' when a plugin cannot be found.
      * @return string
-     * @throws Cake\Core\Exception\MissingPluginException when a plugin path cannot be resolved.
+     * @throws \Cake\Core\Cake\Core\Exception\MissingPluginException when a plugin path cannot be resolved.
```
[the remark this answers is not in the document]
Comment (saeideng) [reply src/Core/PluginCollection.php]: 4.x branch now is correct

**Pull request merged**